test/e2e/framework/util.go

```diff
@@ -450,6 +450,16 @@ func SkipUnlessServerVersionGTE(v *utilversion.Version, c discovery.ServerVersio
 	}
 }
 
+func SkipUnlessServerVersionLT(v *utilversion.Version, c discovery.ServerVersionInterface) {
```

Comment:
writing testcases bounded by future versions is weird enough that I don't want to encourage it by adding these framework functions. prefer something like this:

```go
	BeforeEach(func() {
		removedAlphaTableTransform, err := framework.ServerVersionGTE(removedAlphaTableTransformVersion, f.ClientSet.Discovery())
		Expect(err).NotTo(HaveOccurred())
		if removedAlphaTableTransform {
			framework.Skipf("...")
		}
	})
```

Reply:
Makes sense, updated PR to match this. Thanks